<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainToolProperties</class>
 <widget class="QWidget" name="TerrainToolProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="TilesetTypeLabel">
     <property name="text">
      <string>Terrain type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="TerrainTypeComboBox">
     <item>
      <property name="text">
       <string>Accessible</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inaccessible</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Label_2D">
     <property name="text">
      <string>2D Terrain:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="Tileset2DFileButton">
     <property name="text">
      <string>2D Terrain</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/tool-icons/res/browse-icon.png</normaloff>:/tool-icons/res/browse-icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="Label_3D">
     <property name="text">
      <string>3D Terrain:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="Tileset3DFileButton">
     <property name="text">
      <string>3D Terrain</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/tool-icons/res/browse-icon.png</normaloff>:/tool-icons/res/browse-icon.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
